app/main/views/g_cloud.py

```diff
@@ -179,6 +186,7 @@ def search_services():
         abort(404)
 
     search_api_response = search_api_client.search_services(
+        index=framework['framework'],  # TODO: change this to framework['slug'], also search_presenters.py:~132
```

Comment:
We're ready to do this now I think? If not let's make a plan (he said going on hol)

Reply:
Yep I think we are.